src/App.jsx

```diff
@@ -1,40 +1,44 @@
-import React from 'react';
-import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import 'bulma/css/bulma.css';
+import { useState } from 'react';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+export const App = () => {
+  const getCurrentTab = (tabID) => {
```

Comment:
to optimize the code and not have to go through the array every time you change the ID
you can make an object where the key is the ID and the value of the object is tab
likes this:
```
const tabsById = tabs.reduce((accumulator, currentValue) => {
  accumulator[currentValue.id] = currentValue;
  return accumulator;
}, {});
```
and final:
`const selectedTab = tabsById[selectedTabId] || tabs[0];`

Reply:
better will be:
`const selectedTab = tabsById[selectedTabId] || null;`
because this way we will find and solve the error faster if there is something wrong with the data
`{selectedTab && <h1 className="title">Selected tab is ${selectedTab.title}</h1>}`
if we don't see a title, it's a sign that there's something wrong with it
the backend can return broken data, so this approach allows you to quickly find problems and solve them
this approach is called [Garbagein, garbage out](https://en.wikipedia.org/wiki/Garbage_in,_garbage_out)